build-tools/automation/azure-pipelines.yaml

```diff
@@ -151,6 +151,13 @@ stages:
       displayName: set JI_JAVA_HOME
 
     - template: yaml-templates\use-dot-net.yaml
+      parameters:
+        remove_dotnet: true
+
+    # xabuild still depends on .NET Core 3 or earlier
```

Comment:
It might be worth using the `UseDotNet` task here, as I think it can pull from a cache?

Reply:
It looks like this may only work if you install into `$(Agent.ToolsDirectory)`. I think I'll stick with our template as it also includes some retries, and this extra step only took 21 seconds in the PR run.